components/form/FPhoneInput.vue

```diff
@@ -6,13 +6,14 @@ FField.FPhoneInput(
 )
   FMenu(
     :options="countries"
-    v-model:is-open="isMenuOpen"
+    @toggle="newValue => (isMenuOpen = newValue)"
     v-model="countryCode"
     width="100"
     :color="optionsMenuColor"
     :text-color="optionTextColor"
+    :disabled="disabled"
```

Comment:
j'ai plutôt tendance à mettre les évènements en bas après les props, (c'est d'ailleurs une bonne pratique)
```suggestion
    v-model="countryCode"
    width="100"
    :color="optionsMenuColor"
    :text-color="optionTextColor"
    :disabled="disabled"
    @toggle="newValue => (isMenuOpen = newValue)"
```